components/statistics/SankeyChartBox/index.tsx

```diff
@@ -0,0 +1,84 @@
+import dynamic from 'next/dynamic';
+import { getSankeyChartData } from './SankeyChartBox.helper';
+import { PropsType } from './SankeyChartBox.type';
+import React from 'react';
+import { barChartColors } from '../ChartBarBox/ChartBarBox.helper';
+
+const SankeyChart = dynamic(() => import('./SankeyChart'), {
+  ssr: false,
+});
+
+function SankeyChartBox(props: PropsType) {
+  const { topSourcePath, blockchainDataMap } = props;
+
+  const { data, labels, priority, topDestinations, topSources } =
+    getSankeyChartData(topSourcePath, blockchainDataMap);
+
+  // map structure for assign color for each blockchain.
+  const blockchainColorMap = new Map<string, string>();
+  const barChartColorLength = barChartColors.length;
+  let colorIndex = 0;
+
+  // first assign color to all top sources blockchain.
+  topSources.forEach((sourceItem) => {
+    blockchainColorMap.set(
+      sourceItem,
+      barChartColors[colorIndex % barChartColorLength],
+    );
+    colorIndex += 1;
+  });
+
+  // second assign color to all top destinations blockchain that not contain in source list.
+  topDestinations
+    .filter((destinationItem) => !topSources.includes(destinationItem))
+    .forEach((destinationItem) => {
+      blockchainColorMap.set(
+        destinationItem,
+        barChartColors[colorIndex % barChartColorLength],
+      );
+      colorIndex += 1;
+    });
+
+  return (
+    <div className="w-full bg-baseForeground px-15 py-20 md:p-35 mt-25 rounded-soft md:rounded-normal ">
+      <div className="text-28 font-semibold">
+        Transfer volume among top source & destination chains
+      </div>
+      <div className="w-full flex items-start mt-20">
+        <div className="w-full h-full h-[486px] px-20">
```

Comment:
I think the left padding is extra here.
```suggestion
        <div className="w-full h-full h-[486px] pr-20">
```

![Screenshot from 2024-05-08 06-29-07](https://github.com/rango-exchange/explorer/assets/99180934/9e5f3cbf-b5fa-4b12-8cf8-7eb68bd9246c)